### ISSUE TITLE
crt filter

### ISSUE THREAD

**Issue by HeladoDeBrownie** [HeladoDeBrownie/Nexus#111]
a crt filter would add more visual character while also possibly allowing certain pixel techniques to shine better. it should be disableable for accessibility reasons
